synthesizer/event.cpp

```diff
@@ -396,7 +396,7 @@ void EventMap::fixupMIDI()
             if (it->second.type() == ME_NOTEON) {
                   unsigned short np = info[it->second.channel()].nowPlaying[it->second.pitch()];
                   if (it->second.velo() == 0) {
-                        /* already off or still playing? */
+                        /* already off (should not happen) or still playing? */
```

Comment:
Checking the unlikely condition first? Reverting the order might be an optimization then

Reply:
No, “already off” _must_ be checked first, because if it does not trigger, we immediately decrement before the other check.

Reply:
Ah, yes, the pre-increment, missed that